operators/pkg/utils/test/integration.go

```diff
@@ -41,3 +46,29 @@ func RunWithK8s(m *testing.M, crdPath string) {
 	t.Stop()
 	os.Exit(code)
 }
+
+// StartManager sets up a manager and controller to perform reconciliations in background.
+// It must be stopped by calling the returned function.
+func StartManager(t *testing.T, addToMgrFunc func(manager.Manager, operator.Parameters) error, parameters operator.Parameters) (k8s.Client, func()) {
```

Comment:
This comes from https://github.com/elastic/cloud-on-k8s/pull/906, should disappear from the diff once #906 is merged.